src/main/java/study/calculator/StringCalculator.java

```diff
@@ -14,16 +15,16 @@ public static int calculate(String input) {
         }
 
         String[] numbers = split(input);
+        int[] parseNumbers = parseStringToInteger(numbers);
 
-        int sum = sum(numbers);
-        return sum;
+        return sum(parseNumbers);
```

Comment:
```suggestion
        return sum(parseStringToInteger(split(input)));
```
메서드 체인을 이용해 구현해 보는 것은 어떨까?